src/main/java/epicsquid/roots/entity/ritual/EntityRitualFlowerGrowth.java

```diff
@@ -75,8 +73,7 @@ private boolean generateFlower(BlockPos pos) {
       }
       IBlockState blockState = world.getBlockState(pos.down());
       Block block = blockState.getBlock();
-
-      IItemStack soil = new MCItemStack(new ItemStack(block, block.getMetaFromState(blockState)));
+      IItemStack soil = new MCItemStack(new ItemStack(block, 1, block.getMetaFromState(blockState)));
```

Comment:
This is problematic as it creates a hard dependency on CraftTweaker. Ideally the allowed blocks should be stored as a list of standard Minecraft ingredients (`CraftTweakerMC::getIngredient` from an IIngredient) when the recipe is created.

As it stands now, I think this will crash if someone loads it in an instance without CraftTweaker.